lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -207,6 +207,33 @@ def link_children(filters = {})
         end
       end
 
+      # This method plucks the attributes of your ancestors' ancestors, along with the
```

Comment:
That's a curious use of "your" as opposed to "a policy machine's". In the interest of not plucking my ancestors, mild preference for changing.